src/lapack/dec/q/qr2ut/front/flamec/test/flash_sm/time_QR2_UT.c

```diff
@@ -58,7 +58,7 @@ void time_QR2_UT(
       case FLA_ALG_REFERENCE:
 //FLA_Obj_show( "A_flat:", A_flat, "%12.4e", "" );
         //FLA_QR_UT( A_flat, t, T_flat );
-        FLA_QR2_UT( B_flat,
```

Comment:
Given that this is the FLA_ALG_REFERENCE path, no FLASH substitution necessary

Reply:
FLA_QR2_UT was giving an undefined reference. It looks like it was removed at some point. Should this be something other than FLASH_QR2_UT?

Reply:
Does `FLA_QR2_UT_internal` work?